dataGenerator.js

```diff
@@ -88,182 +52,36 @@ while (numberOfReviews < numberToWrite) {
-    } else if (numberOfReviews / numberToWrite < .4) {
-      reviews4 += `${numberOfReviews + 1},${product_id + 1},${random(1, 6)},${date.toString().slice(0, 24)},${faker.random.words()},${faker.lorem.paragraph()},${(Math.random() >= .3)},${false},${faker.name.findName()},${faker.internet.email()},${"none"},${random(0, 30)}\n`
-    } else if (numberOfReviews / numberToWrite < .5) {
-      reviews5 += `${numberOfReviews + 1},${product_id + 1},${random(1, 6)},${date.toString().slice(0, 24)},${faker.random.words()},${faker.lorem.paragraph()},${(Math.random() >= .3)},${false},${faker.name.findName()},${faker.internet.email()},${"none"},${random(0, 30)}\n`
-    } else if (numberOfReviews / numberToWrite < .6) {
-      reviews6 += `${numberOfReviews + 1},${product_id + 1},${random(1, 6)},${date.toString().slice(0, 24)},${faker.random.words()},${faker.lorem.paragraph()},${(Math.random() >= .3)},${false},${faker.name.findName()},${faker.internet.email()},${"none"},${random(0, 30)}\n`
-    } else if (numberOfReviews / numberToWrite < .7) {
-      reviews7 += `${numberOfReviews + 1},${product_id + 1},${random(1, 6)},${date.toString().slice(0, 24)},${faker.random.words()},${faker.lorem.paragraph()},${(Math.random() >= .3)},${false},${faker.name.findName()},${faker.internet.email()},${"none"},${random(0, 30)}\n`
-    } else if (numberOfReviews / numberToWrite < .8) {
-      reviews8 += `${numberOfReviews + 1},${product_id + 1},${random(1, 6)},${date.toString().slice(0, 24)},${faker.random.words()},${faker.lorem.paragraph()},${(Math.random() >= .3)},${false},${faker.name.findName()},${faker.internet.email()},${"none"},${random(0, 30)}\n`
-    } else if (numberOfReviews / numberToWrite < .9) {
-      reviews9 += `${numberOfReviews + 1},${product_id + 1},${random(1, 6)},${date.toString().slice(0, 24)},${faker.random.words()},${faker.lorem.paragraph()},${(Math.random() >= .3)},${false},${faker.name.findName()},${faker.internet.email()},${"none"},${random(0, 30)}\n`
-    } else if (numberOfReviews / numberToWrite <= 1) {
-      reviews10 += `${numberOfReviews + 1},${product_id + 1},${random(1, 6)},${date.toString().slice(0, 24)},${faker.random.words()},${faker.lorem.paragraph()},${(Math.random() >= .3)},${false},${faker.name.findName()},${faker.internet.email()},${"none"},${random(0, 30)}\n`
-    }
+    reviews += `${numberOfReviews + 1},${product_id + 1},${random(1, 6)},${date.toString().slice(0, 24)},${faker.random.words()},${faker.lorem.paragraph()},${(Math.random() >= .3)},${false},${faker.name.findName()},${faker.internet.email()},${"none"},${random(0, 30)}\n`
 
     // generate values for this review's characteristics + add to csv
     for (var j = 0; j < products[product_id].characteristics.length; j++) {
```

Comment:
isn't products[product_id].characteristics.length = 4? can you either write 4  (with a short comment about about why 4) or set up a variable with a simpler name (charLength) so that this is more readable. Your future self will thank you.